<launch>
<!--
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="/dev/ttyACM0"/>
    <param name="baud" value="1000000"/>
  </node>
  -->
  <node pkg="robot_base" type="odom.py" name="odom_node"/>
  <node pkg="grap_moves" type="move.py" name="move_robot_service"/>
  <!--
<node name="hokuyo_back" pkg="hokuyo_node" type="hokuyo_node" respawn="false" output="screen">
    <param name="calibrate_time" type="bool" value="true"/> 
    <param name="port" type="string" value="/dev/ttyACM2"/>   
    <param name="intensity" type="bool" value="false"/>
    <remap from="scan" to="scan_back"/>
    <param name="frame_id" value="laser_hokuyo_2"/>    
    <param name="min_ang" type="double" value="-1.75"/>
    <param name="max_ang" type="double" value="1.75"/>
  </node>  

  <node name="hokuyo_front" pkg="hokuyo_node" type="hokuyo_node"  respawn="false" output="screen">
    <param name="calibrate_time" type="bool" value="true"/> 
    <param name="port" type="string" value="/dev/ttyACM1"/>   
    <param name="intensity" type="bool" value="false"/>
  	<remap from="scan" to="scan_front"/>
    <param name="frame_id" value="laser_hokuyo"/>    
    <param name="min_ang" type="double" value="-1.7"/>
    <param name="max_ang" type="double" value="1.7"/>
    <param name="cluster" type="double" value="3"/>
  </node> 
  -->
   <node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
          <param name="destination_frame" value="summit_base_link"/>
    <param name="cloud_destination_topic" value="/merged_cloud"/>
    <param name="scan_destination_topic" value="/scan"/>
                <param name="laserscan_topics" value ="/scan_back /scan_front" /> 
                <param name="angle_min" value="-3.14"/>
                <param name="angle_max" value="3.14"/>
                <param name="angle_increment" value="0.0058"/>
                <param name="scan_time" value="0.0333333"/>
                <param name="range_min" value="0.03"/>
                <param name="range_max" value="50.0"/>

  </node>

  <include file="$(find navigation)/launch/ukf_template.launch"/> 
</launch>
